src/MarkerCluster.js

```diff
@@ -44,7 +44,9 @@ L.MarkerCluster = L.Marker.extend({
 	},
 
 	//Zoom to the minimum of showing all of the child markers, or the extents of this cluster
-	zoomToBounds: function () {
+	zoomToBounds: function (fitBoundsOptions) {
+        fitBoundsOptions = fitBoundsOptions || {};
```

Comment:
I don't think this is necessary? We'll just pass undefined through which is totally fine.